tests/stests/workloads/delete_workload_podman_kube.robot

```diff
@@ -39,7 +39,7 @@ Test Ankaios Podman delete kube workload
     # Actions
     When user triggers "ank delete workload hello-k8s"
     # Asserts
-    Then the workload "hello-k8s" shall not exist
```

Comment:
We can check the execution state here too and expect removed.

Same goes for the tests below. Please check them again.

Reply:
Fixed